src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -92,6 +91,16 @@ public boolean handle(int code) {
             });
         }
     }
+    
+    private void setupSecurity(Settings settings, Environment environment) throws Exception {
+        ESLogger logger = Loggers.getLogger(Bootstrap.class);
+        if (settings.getAsBoolean("security.enabled", true)) {
+            Security.configure(environment);
+            logger.info("security enabled");
+        } else {
+            logger.warn("security disabled");
+        }
```

Comment:
This is officially back and forth at this point. I started at info and went to warn based on feedback. Guys get your shit together, im not playing this game.